cimloader/databases/blazegraph.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import subprocess
+
+from cimloader.databases import ConnectionInterface, QueryResponse
+from SPARQLWrapper import JSON, POST, SPARQLWrapper
+
+_log = logging.getLogger(__name__)
+
+class BlazegraphConnection(ConnectionInterface):
```

Comment:
BlazegraphConnection needs to be changed remove potential ambiguity with cimgraph.database.blazegraph.BlazegraphConnection.